packages/backend-core/src/queue/inMemoryQueue.ts

```diff
@@ -83,6 +87,12 @@ class InMemoryQueue implements Partial<Queue> {
   async process(concurrencyOrFunc: number | any, func?: any) {
     func = typeof concurrencyOrFunc === "number" ? func : concurrencyOrFunc
     this._emitter.on("message", async message => {
+      // For the purpose of testing, don't trigger cron jobs immediately.
+      // Require the test to trigger them manually with timestamps.
+      if (message.opts?.repeat != null) {
+        return
+      }
```

Comment:
It used to be that when you called `config.api.application.publish()`, all crons were triggered straight away. This was odd behaviour to write tests against, so I decided to defer all cron triggers to be done manually in tests because it's less spooky.